configuration/openmrs/apps/clinical/app.json

```diff
@@ -512,6 +512,34 @@
         "Cambodia_Malaria Rapid Test" : {
           "disableAddNotes": true
         },
+        "Cambodia_Date of Delivery" : {
+          "disableAddNotes": true
+        },
+        "Cambodia_Newborn Sex" : {
+          "disableAddNotes": true
+        },
+        "Cambodia_Newborn Weight" : {
+          "disableAddNotes": true
+        },
+        "Cambodia_PNC Problems During Delivery" : {
+          "disableAddNotes": true
+        },
+        "Cambodia_PNC First Visit" : {
+          "disableAddNotes": true
+        },
+        "Cambodia_Maternal Care" : {
+          "disableAddNotes": true
```

Comment:
As per a comment in TLC-59 ([comment](https://mekomsolutions.atlassian.net/browse/TLC-59?focusedCommentId=34863&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-34863)), I suggest you add `multiselect: true` to some fields.
Could you change this?

Note: I think yo can just change it on your fork and push the changes, then the PR will be updated too